plugin/wordepress/wordepress.php

```diff
@@ -2,88 +2,99 @@
 /*
 Plugin Name: Weaveworks Wordepress
 Description: Host technical documentation in WordPress
-Version: 1.0.0
+Version: 1.1.0
 Author: Adam Harrison
 */
 
-// Register the `document` post type with the REST API
-add_action( 'init', 'add_document_cpt_rest_support', 25 );
-function add_document_cpt_rest_support() {
+function wordepress_init_post_type( $post_type_name ) {
     global $wp_post_types;
 
-    $post_type_name = 'documentation';
     if( isset( $wp_post_types[ $post_type_name ] ) ) {
         $wp_post_types[$post_type_name]->show_in_rest = true;
         $wp_post_types[$post_type_name]->rest_base = $post_type_name;
         $wp_post_types[$post_type_name]->rest_controller_class = 'WP_REST_Posts_Controller';
     }
 }
 
-add_action( 'rest_api_init', 'slug_register_document' );
-function slug_register_document() {
+// Register the `document` post type with the REST API
+add_action( 'init', function () { wordepress_init_post_type ( 'documentation' ); }, 100);
+
+// Register the `guide` post type with the REST API
+add_action( 'init', function() { wordepress_init_post_type ( 'tutorials' ); }, 100);
```

Comment:
Yeah, also panned to clean this up – the reason we have to use word `tutorial` is because there are `guides` in WP already. We need some kind of migration plan and I have no idea what's best, a new name seemed reasonable.